restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/Conference.java

```diff
@@ -95,6 +108,8 @@
     private Sid sid;
     private final List<ActorRef> calls;
     private final List<ActorRef> observers;
+    //list of callApiClients created by Conference
+    private List<ActorRef> callApiClients;
```

Comment:
@maria-farooq There is no need to keep a list of CallApiClients, this is redundant and will consume memory for no reason. See my comment later for the `onDownloaderResponse` method